keras2onnx/_tf_utils.py

```diff
@@ -81,7 +81,7 @@ def tf_attrs_to_onnx(node):
         if s_.startswith('T'):  # all T starts attr is TF internal.
             continue
         v = node.get_attr(s_)
-        if isinstance(v, tensorflow.dtypes.DType):
+        if hasattr(tensorflow.dtypes, 'DType') and isinstance(v, tensorflow.dtypes.DType):
```

Comment:
For python 3.5 CI build, we see `AttributeError: module 'tensorflow.dtypes' has no attribute 'DType'` for this newly added UT.